modules/widgets/simple-payments/admin-warning.php

```diff
@@ -2,11 +2,11 @@
 	<p>
 		<?php
 			$support_url = ( defined( 'IS_WPCOM' ) && IS_WPCOM )
-				? 'https://support.wordpress.com/simple-payments/'
-				: 'https://jetpack.com/support/simple-payment-button/';
+				? 'https://wordpress.com/support/pay-with-paypal-button/'
```

Comment:
We may want to update the doc permalink and this url to `https://wordpress.com/support/pay-with-paypal`